requirements.txt

```diff
@@ -1,8 +1,10 @@
-aiohttp==2.2.5
-async-timeout==1.2.1
-chardet==3.0.4
--e git+https://github.com/Rapptz/discord.py@rewrite#egg=discord.py
--e git+https://github.com/strinking/python-duckduckgo@async#egg=duckduckgo
-multidict==3.1.3
-websockets==3.4
-yarl==0.12.0
+PyYAML>=3.10
+aiohttp>=2.2.0
+async-timeout>=1.2.0
+chardet>=3.0.0
+multidict>=3.1.0
+ratelimit>=1.4.0
+websockets>=3.0
+yarl>=0.12.0
+git+https://github.com/Rapptz/discord.py@rewrite#egg=discord.py[voice]
+-e git+https://github.com/strinking/python-duckduckgo@master#egg=duckduckgo-async
```

Comment:
again, thoughts about using pipenv instead of "the old way"?

Reply:
pipenv seems neat, but I've never used it before. if it's integrated and documented well I have no qualms with it.

however, that support should probably happen in another PR. this one is getting pretty big

Reply:
agreed